packages/truffle-box/box.js

```diff
@@ -111,17 +110,16 @@ const Box = {
       tmp.setGracefulCleanup();
     }
 
-    tmp.dir({ unsafeCleanup }, function(err, dir) {
+    tmp.dir({ unsafeCleanup }, (err, dir) => {
```

Comment:
If you are feeling ambitious, I would be interested in having the synchronous version of this method used :) If you are not interested that is also fine. Check it out --> https://www.npmjs.com/package/tmp